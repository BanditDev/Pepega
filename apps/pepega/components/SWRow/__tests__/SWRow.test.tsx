/* eslint-env jest */
import { mount, render } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import React from 'react';
import renderer from 'react-test-renderer';

import { SWRow } from '../';

describe('SWRow', () => {
  it('should render correctly', () => {
    const wrapper = render(<SWRow title="Title">SWRow</SWRow>);
    expect(wrapper).toMatchSnapshot();
  });

  it('should mount correctly', () => {
    expect(() =>
      renderer.create(<SWRow title="Title">SWRow</SWRow>)
    ).not.toThrow();
  });
});
