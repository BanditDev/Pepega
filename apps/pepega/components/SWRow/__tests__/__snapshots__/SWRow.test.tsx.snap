// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SWRow should render correctly 1`] = `
initialize {
  "0": Object {
    "attribs": Object {
      "class": "SWRow__Box-sc-1atccaz-0 huaKOo",
    },
    "children": Array [
      Object {
        "attribs": Object {
          "class": "SWRow__SRowLeft-sc-1atccaz-1 leadox",
        },
        "children": Array [
          Object {
            "attribs": Object {
              "class": "SWRow__SRowTitle-sc-1atccaz-2 duUbHA",
            },
            "children": Array [
              Object {
                "data": "Title",
                "next": null,
                "parent": [Circular],
                "prev": null,
                "type": "text",
              },
            ],
            "name": "div",
            "namespace": "http://www.w3.org/1999/xhtml",
            "next": null,
            "parent": [Circular],
            "prev": null,
            "type": "tag",
            "x-attribsNamespace": Object {
              "class": undefined,
            },
            "x-attribsPrefix": Object {
              "class": undefined,
            },
          },
        ],
        "name": "div",
        "namespace": "http://www.w3.org/1999/xhtml",
        "next": Object {
          "attribs": Object {
            "class": "SWRow__SRowRight-sc-1atccaz-4 hZaBOJ",
          },
          "children": Array [
            Object {
              "attribs": Object {
                "class": "SWRow__SRowSwitch-sc-1atccaz-5 faxFnS",
              },
              "children": Array [
                Object {
                  "attribs": Object {
                    "class": "Switch__Box-sc-1gdktzj-0 fuQGsd",
                  },
                  "children": Array [
                    Object {
                      "attribs": Object {
                        "class": "Switch__Slider-sc-1gdktzj-1 dZYeBy",
                      },
                      "children": Array [],
                      "name": "div",
                      "namespace": "http://www.w3.org/1999/xhtml",
                      "next": null,
                      "parent": [Circular],
                      "prev": null,
                      "type": "tag",
                      "x-attribsNamespace": Object {
                        "class": undefined,
                      },
                      "x-attribsPrefix": Object {
                        "class": undefined,
                      },
                    },
                  ],
                  "name": "div",
                  "namespace": "http://www.w3.org/1999/xhtml",
                  "next": null,
                  "parent": [Circular],
                  "prev": null,
                  "type": "tag",
                  "x-attribsNamespace": Object {
                    "class": undefined,
                  },
                  "x-attribsPrefix": Object {
                    "class": undefined,
                  },
                },
              ],
              "name": "div",
              "namespace": "http://www.w3.org/1999/xhtml",
              "next": null,
              "parent": [Circular],
              "prev": null,
              "type": "tag",
              "x-attribsNamespace": Object {
                "class": undefined,
              },
              "x-attribsPrefix": Object {
                "class": undefined,
              },
            },
          ],
          "name": "div",
          "namespace": "http://www.w3.org/1999/xhtml",
          "next": null,
          "parent": [Circular],
          "prev": [Circular],
          "type": "tag",
          "x-attribsNamespace": Object {
            "class": undefined,
          },
          "x-attribsPrefix": Object {
            "class": undefined,
          },
        },
        "parent": [Circular],
        "prev": null,
        "type": "tag",
        "x-attribsNamespace": Object {
          "class": undefined,
        },
        "x-attribsPrefix": Object {
          "class": undefined,
        },
      },
      Object {
        "attribs": Object {
          "class": "SWRow__SRowRight-sc-1atccaz-4 hZaBOJ",
        },
        "children": Array [
          Object {
            "attribs": Object {
              "class": "SWRow__SRowSwitch-sc-1atccaz-5 faxFnS",
            },
            "children": Array [
              Object {
                "attribs": Object {
                  "class": "Switch__Box-sc-1gdktzj-0 fuQGsd",
                },
                "children": Array [
                  Object {
                    "attribs": Object {
                      "class": "Switch__Slider-sc-1gdktzj-1 dZYeBy",
                    },
                    "children": Array [],
                    "name": "div",
                    "namespace": "http://www.w3.org/1999/xhtml",
                    "next": null,
                    "parent": [Circular],
                    "prev": null,
                    "type": "tag",
                    "x-attribsNamespace": Object {
                      "class": undefined,
                    },
                    "x-attribsPrefix": Object {
                      "class": undefined,
                    },
                  },
                ],
                "name": "div",
                "namespace": "http://www.w3.org/1999/xhtml",
                "next": null,
                "parent": [Circular],
                "prev": null,
                "type": "tag",
                "x-attribsNamespace": Object {
                  "class": undefined,
                },
                "x-attribsPrefix": Object {
                  "class": undefined,
                },
              },
            ],
            "name": "div",
            "namespace": "http://www.w3.org/1999/xhtml",
            "next": null,
            "parent": [Circular],
            "prev": null,
            "type": "tag",
            "x-attribsNamespace": Object {
              "class": undefined,
            },
            "x-attribsPrefix": Object {
              "class": undefined,
            },
          },
        ],
        "name": "div",
        "namespace": "http://www.w3.org/1999/xhtml",
        "next": null,
        "parent": [Circular],
        "prev": Object {
          "attribs": Object {
            "class": "SWRow__SRowLeft-sc-1atccaz-1 leadox",
          },
          "children": Array [
            Object {
              "attribs": Object {
                "class": "SWRow__SRowTitle-sc-1atccaz-2 duUbHA",
              },
              "children": Array [
                Object {
                  "data": "Title",
                  "next": null,
                  "parent": [Circular],
                  "prev": null,
                  "type": "text",
                },
              ],
              "name": "div",
              "namespace": "http://www.w3.org/1999/xhtml",
              "next": null,
              "parent": [Circular],
              "prev": null,
              "type": "tag",
              "x-attribsNamespace": Object {
                "class": undefined,
              },
              "x-attribsPrefix": Object {
                "class": undefined,
              },
            },
          ],
          "name": "div",
          "namespace": "http://www.w3.org/1999/xhtml",
          "next": [Circular],
          "parent": [Circular],
          "prev": null,
          "type": "tag",
          "x-attribsNamespace": Object {
            "class": undefined,
          },
          "x-attribsPrefix": Object {
            "class": undefined,
          },
        },
        "type": "tag",
        "x-attribsNamespace": Object {
          "class": undefined,
        },
        "x-attribsPrefix": Object {
          "class": undefined,
        },
      },
    ],
    "name": "div",
    "namespace": "http://www.w3.org/1999/xhtml",
    "next": null,
    "parent": null,
    "prev": null,
    "root": Object {
      "attribs": Object {},
      "children": Array [
        [Circular],
      ],
      "name": "root",
      "namespace": "http://www.w3.org/1999/xhtml",
      "next": null,
      "parent": null,
      "prev": null,
      "type": "root",
      "x-attribsNamespace": Object {},
      "x-attribsPrefix": Object {},
    },
    "type": "tag",
    "x-attribsNamespace": Object {
      "class": undefined,
    },
    "x-attribsPrefix": Object {
      "class": undefined,
    },
  },
  "_root": [Circular],
  "length": 1,
  "options": Object {
    "decodeEntities": true,
    "normalizeWhitespace": false,
    "withDomLvl1": true,
    "xml": false,
  },
}
`;
